### DISCUSSION TITLE
巅峰杯物品识别报错: UnicodeDecodeError

### DISCUSSION THREAD

**Discussion by mtt-konan** [NateScarlet/auto-derby#280] [Q&A] [answered]
![bug3](https://user-images.githubusercontent.com/102540842/161271770-8e194640-a5e0-4e22-ba64-f00166ce6f75.png)

版本auto-derby-1.19.0，不是一开始就报错，而是使用了一段时间才报错。
我以为是dvd物品识别错误，买了其他东西刷掉之后还是一进这个界面就报错，不带插件，重启环境也没用，下面是问题日志：

[bug3.txt](https://github.com/NateScarlet/auto-derby/files/8398092/bug3.txt)

**Comment by NateScarlet** [comment, marked as answer]
你这个是带了插件的日志

看一下直接 `py -3.8 auto_derby/web/page_example.py` 会不会出错

我觉得很有可能是你用中文路径导致的

**Reply by mtt-konan** [reply]
最近重装了vscode的python环境，没想到也影响到了。。应该是python中文路径的问题，我重装python至英文路径试试

**Reply by mtt-konan** [reply]
我排除了中文路径的问题，还是报相同的错误

![Snipaste_2022-04-02_01-07-06](https://user-images.githubusercontent.com/102540842/161310133-d88393dd-2318-4b5a-a9c8-198cfbc75211.png)

**Reply by NateScarlet** [reply]
是不是你电脑主机名有中文 这个我查了下应该是 [python 的 bug](https://bugs.python.org/issue26227)

而且已经被标记为解决 说不定你更新下 python 版本就行

**Reply by mtt-konan** [reply]
解决了，办法就是把python从3.8.3升到3.8.10，再高版本我也没试，怕有些地方不兼容

**Reply by NateScarlet** [reply]
我用 3.8.0 反而没这个问题

**Reply by mtt-konan** [reply]
本来用的好好的，装了vscode的python调试插件就出这个bug了，之后重装vscode和python都没用，不过现在至少问题解决了。。
